package.json

```diff
@@ -12,20 +12,23 @@
   "dependencies": {
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
-    "@graasp/apps-query-client": "3.4.1",
-    "@graasp/sdk": "3.10.0",
-    "@graasp/ui": "3.0.0",
+    "@graasp/apps-query-client": "https://github.com/graasp/graasp-apps-query-client.git#main",
```

Comment:
Use the newest version of apps-query-client? Why the change?
But I know recent changes might have broken a lot of things, so if it results in too many changes, use a previous version.